plugins/cody-chat/src/com/sourcegraph/cody/chat/agent/CodyManager.java

```diff
@@ -64,15 +67,6 @@ public void withAgent(Consumer<CodyAgent> action) {
     withAgent(OnFailure.LOG, action);
   }
 
-  public void onConfigChange(ExtensionConfiguration config) {
-    this.config = config;
-    CodyLogger.onConfigChange(config);
-    if (agentHolder.get() != null) {
-      // Notify agent about config change
-      withAgent(OnFailure.LOG, agent -> agent.server.extensionConfiguration_didChange(config));
-    }
-  }
```

Comment:
Why is this removed? The only issue I see is that the CodyLogger.onConfigChange sets the endpoint in the log which is a nice-to-have.

Reply:
Oh I see. Somewhere else we stopped calling this method entirely, so we're never setting the endpoint.

Reply:
The endpoint is set from the webview now. Previously, we set it from the native Eclipse UI to manage account endpoint/access_token pairs.